docker-compose.yml

```diff
@@ -9,7 +9,6 @@ web:
   environment:
   - VIRTUAL_PORT=${ESM_HOST_PORT}
   - VIRTUAL_HOST=${ESM_VIRTUAL_HOST}
-  - SEED_MEM=${SEED_MEM}
   - MEM=${MEM}
```

Comment:
As above, do we need a reference to MEM?

Reply:
Going to leave this